### DISCUSSION TITLE
Can toolbox process continuous predictor?

### DISCUSSION THREAD

**Discussion by Ycx2000** [unfoldtoolbox/unfold#137] [Q&A] [answered]
Hi dear Benedikt,

Thank you for developing this fantastic toolbox! I'm a postgraduate student, and I'm curious about whether this toolbox can be used to process continuous predictors. Specifically, when an event occurs, I would like to compute a corresponding continuous variable. I'm interested in understanding how the beta value changes in relation to the variable's values.

As far as I know, it seems that the toolbox currently allows the categorization of predictors into several conditions. If my understanding is correct, could you please provide any suggestions on how to handle continuous variables ?

I hope I have expressed my question accurately. Thanks for your valuable contribution!

Best regards,
Eve

**Comment by olafdimigen** [comment, marked as answer]
Hi Eve,

that is absolutely possible. You can define intercept terms, categorical predictors (factors), continuous linear predictors, or continuous nonlinear (spline) predictors and these can of course also be combined. A continous predictors would be a tone displayed at variable loudness levels. You find a tutorial (for the case of nonlinear continuous predictors) here: https://www.unfoldtoolbox.org/toolbox-tut03.html

A different issue are _time_-continous predictors, that is, predictors that extend through time (e.g. a long audio waveform rather than a discrete event of a single tone). i dont think you mean this but you find the tutorial on that here: https://www.unfoldtoolbox.org/toolbox-tut08.html

Cheers, Olaf

**Reply by Ycx2000** [reply]
Hi Olaf, 

Thanks for your reply!
I do mean the first situation, and I've found the solution in the tutorial. Sorry for asking such an obvious question ~

Best regards，Eve
